src/Nethermind/Nethermind.TxPool.Test/TxPoolTests.Blobs.cs

```diff
@@ -490,5 +491,61 @@ public void should_calculate_size_of_blob_tx_correctly(int numberOfBlobs, int ex
                 .TestObject;
             blobTx.GetLength().Should().Be(expectedLength);
         }
+
+        [Test]
+        public async Task should_add_processed_txs_to_db()
```

Comment:
no test scenarios for block reorgs? I think if reorgs are super hard to test in devnets it would be nice to add unit tests for a few reogrs scenarios